x-pack/test/apm_api_integration/tests/services/top_services.ts

```diff
@@ -44,187 +53,242 @@ export default function ApiTest({ getService }: FtrProviderContext) {
   );
 
   registry.when(
-    'APM Services Overview with a basic license when data is loaded',
-    { config: 'basic', archives: [archiveName] },
+    'APM Services Overview with a basic license when data is generated',
+    { config: 'basic', archives: ['apm_mappings_only_8.0.0'] },
     () => {
       let response: {
         status: number;
         body: APIReturnType<'GET /internal/apm/services'>;
       };
 
-      let sortedItems: typeof response.body.items;
+      const range = timerange(new Date(start).getTime(), new Date(end).getTime());
+      const transactionInterval = range.interval('1s');
+      const metricInterval = range.interval('30s');
+
+      const multipleEnvServiceProdInstance = service(
+        'multiple-env-service',
+        'production',
+        'go'
+      ).instance('multiple-env-service-production');
+
+      const multipleEnvServiceDevInstance = service(
+        'multiple-env-service',
+        'development',
+        'go'
+      ).instance('multiple-env-service-development');
```

Comment:
nit: it looks like there's just two instances: a dev and a prod:

```suggestion
      const devInstance = service(
        'multiple-env-service',
        'production',
        'go'
      ).instance('multiple-env-service-production');

      const prodInstance = service(
        'multiple-env-service',
        'development',
        'go'
      ).instance('multiple-env-service-development');
```

Reply:
oh, I see there's also a `metricOnlyInstance`